docs/components/connectors/out-of-the-box-connectors/power-automate.md

```diff
@@ -0,0 +1,199 @@
+
+#### Input
+
+For this section, you must fill out the following field:
+
+1. **Flow session ID**: Unique identifier for entity instances. You can find this in the response of the 'Trigger a flow run' method.
+
+#### Output
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. It comes with a pre-filled value of `= {statusCode: response.body.statuscode}`.
+
+### Get flow outputs
+
+The operation **Get flow outputs** returns the output of the triggered Power Automate flow.
+
+#### Configuration
+
+For this section, you must fill out the following field:
+
+1. **Organization URI**: This is your dynamics 365 URL. [Visit the docs](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/discovery-service) or [learn about environments](https://learn.microsoft.com/en-us/power-platform/admin/environments-overview#environment-details) to see how to find it.
+
+#### Input
+
+For this section, you must fill out the following field:
+
+1. **Flow session ID**: Unique identifier for entity instances. You can find this in the response of the 'Trigger a flow run' method.
+
+#### Output
+
+You can use an output mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. It comes with a pre-filled value of `= {flowOutputs: response.body}`.
+
+## Appendix
+
+### Using Power Automate Connector best practice
+
+#### Using Webhook connector as callback endpoint
+
+Oftentimes it is desired to continue the process after a Power Automate flow run finished. You can use the **Callback URL** field if you select the **Trigger a flow run** to specify a url which will be called after the flow is finished.
+If you wish to continue the Camunda process or start a new one, the [Webhook Connector](./http-webhook.md) suits very well. For that you need the following steps:
+
+1. Create a [Webhook Connector](./http-webhook.md) and specify the **Webhook ID**.
+2. Create a Power [Automate connector](./power-automate.md), select the **Trigger a flow run** method, fill in the fields.
+
+   If you use Camunda 8 Saas you can use the following pattern for the **Callback URL**: `{zeebe.client.cloud.region}.{zeebeHostUrl}/{zeebe.client.cloud.clusterId}/inbound/{webhookId}`.
+
+   For example: `https://bru-3.connectors.camunda.io/0100101a-11a1-1111-a1a1-1a1a11a11111/inbound/a1aa1aa1-a111-1111-a1aa-1111aa1a1a11`.
+
+   If you use Camunda 8 Saas you can find the zeebe specific values in you cluster details on the _API_ tab under _Client Credentials_. The _webhookId_ is the id you specified in the first step, and the _zeebeHostUrl_ is connectors.camunda.io.
```

Comment:
we are releasing the new copy url function in webmodeler. We should reference here: https://camunda.slack.com/archives/C04PK0YK19A/p1677858199268699. This will be released soon

Reply:
I did not include this yet. We have to include it later but before the release.

Reply:
Will include later, after the release